src/accessibility.js

```diff
@@ -21,22 +28,39 @@ const reduceLines = (lineLength) =>
       : newLine(lines, word);
   };
 
-/**
- * transforms a string into an array of word wrapped lines
- * @param {string} text
- * @param {number} lineLength - max length of a line
- * @returns {string[]}
- */
-export const wordWrap = (text, lineLength) => {
-  return text.split(' ')
-    .filter((e) => e !== '')
-    .reduce(reduceLines(lineLength), [''])
-    .map((e) => e.trim());
-};
+//word wrap transformations
```

Comment:
you should include a space between your `//` and the text of your comment 
```suggestion
// word wrap transformations
```